content/guide/property-system.md

```diff
@@ -0,0 +1,315 @@
+---
+title: Property
+---
+
+The property system in NativeScript is a way to expose a native property when you are exposing a native UI component.
+
+NativeScript provides a basic property system via the `Property` class, which is a simple wrapper around the [Object.defineProperty](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty) with some additional callbacks like `valueChange`, `valueConverter` and `equalityComparer`.
```

Comment:
Within NativeScript, the Property class serves as a fundamental property system, encompassing a straightforward wrapper around the [Object.defineProperty](https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty) function. This class also incorporates supplementary callbacks such as valueChange, valueConverter, and equalityComparer.